css-ecommerce-page/index.html

```diff
@@ -13,35 +13,35 @@
       <h1 class="text-align-center hundred-width">New Arrivals</h1>
     </div>
     <div class="row">
-      <div class="one-fifth">
+      <div class="one-fifth left-right-margin">
         <div class="img-div text-align-center">
           <img src="images/dress1.jpeg" alt="">
         </div>
         <p class="size"><span id="brand">Free People </span>| Size 6</p>
         <p class="old-price"><span id="new-price">$35</span><s> $125 retail</s></p>
       </div>
-      <div class="one-fifth">
+      <div class="one-fifth left-right-margin">
         <div class="img-div text-align-center">
           <img src="images/ysl-bag.jpeg" alt="">
         </div>
         <p class="size"><span id="brand">Yves Saint Laurent </span>| One-Size</p>
         <p class="old-price"><span id="new-price">$200</span><s> $1600 retail</s></p>
       </div>
-      <div class="one-fifth">
+      <div class="one-fifth left-right-margin">
         <div class="img-div text-align-center">
           <img src="images/key.jpeg" alt="">
         </div>
         <p class="size"><span id="brand">Tiffany </span>| One-Size</p>
         <p class="old-price"><span id="new-price">$476</span><s> $2000 retail</s></p>
       </div>
-      <div class="one-fifth">
-        <div class="img-div text-align-center height-245">
-          <img class="object-fit-contain height-245" src="images/lv-bag.jpg" alt="">
+      <div class="one-fifth left-right-margin">
+        <div class="img-div text-align-center height-230">
+          <img class="object-fit-contain height-230" src="images/lv-bag.jpg" alt="">
```

Comment:
Rather than having one weird image height, try to make a style that you can apply to all of them so the code is uniform.

Reply:
Just fixed. Thank you for the feedback, Tim.